packages/docusaurus/src/server/plugins/__tests__/index.test.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import path from 'path';
+
+import {loadContext} from '@docusaurus/core/src/server/index';
```

Comment:
this is the other part of the issue: code from core should use relative imports, not code from `@docusaurus/core` that is the public API for other packages that depends on core

Reply:
`import {loadContext} from '../../index';`